src/program.js

```diff
@@ -447,15 +447,15 @@ Example: $0 --help run.
                   'preference.',
         demand: false,
         requiresArg: true,
-        type: 'string',
+        type: 'array',
```

Comment:
Could you also update this Flow type:

https://github.com/rpl/web-ext/blob/3848c777b290b4c5ed6ad1af0436afe5f952b005/src/firefox/preferences.js#L120

It should now just be `cliPrefs: Array<string>`. I think you may need to update some tests after that.